Error: ENOENT: no such file or directory, open '/tmp/my-app/pkg/yew_app_bg.js'

```
npm run build

> yew-app@0.1.0 build /tmp/my-app
> WASM_PACK_PROFILE=release parcel build static/index.html --public-url ./

🚨  /tmp/my-app/Cargo.toml: ENOENT: no such file or directory, open '/tmp/my-app/pkg/yew_app_bg.js'
Error: ENOENT: no such file or directory, open '/tmp/my-app/pkg/yew_app_bg.js'
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! yew-app@0.1.0 build: `WASM_PACK_PROFILE=release parcel build static/index.html --public-url ./`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the yew-app@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ecarroll/.npm/_logs/2020-06-24T03_02_24_338Z-debug.log
```

--- a/crates/yew-app/Cargo.toml
+++ b/crates/yew-app/Cargo.toml
@@ -18,12 +18,12 @@ crate-type = ["cdylib", "rlib"]
 log = "0.4"
 yew = "0.16.2"
 yew-router = { version="0.13.0", features = ["web_sys"] }
-wasm-bindgen = "0.2, <0.2.61"
+wasm-bindgen = "0.2.63"
 wasm-logger = "0.2.0"
 wee_alloc = "0.4.5"
 
 [dev-dependencies]
-wasm-bindgen-test = "0.3.9"
+wasm-bindgen-test = "0.3.13"
 
 [dev-dependencies.web-sys]
 version = "0.3"

--- a/crates/yew-app/package.json
+++ b/crates/yew-app/package.json
@@ -9,7 +9,7 @@
   },
   "devDependencies": {
     "parcel-bundler": "^1.12.4",
-    "parcel-plugin-wasm.rs": "^1.2.15",
+    "parcel-plugin-wasm.rs": "^1.2.16",
     "sass": "^1.24.2",
     "typescript": "^3.7.4"
   }

